service/src/main/java/org/cbioportal/genome_nexus/service/factory/IsoformAnnotationEnricherFactory.java

```diff
@@ -0,0 +1,34 @@
+package org.cbioportal.genome_nexus.service.factory;
+
+import org.cbioportal.genome_nexus.service.config.AppConfig;
+import org.cbioportal.genome_nexus.service.EnsemblService;
+import org.cbioportal.genome_nexus.service.OncokbService;
+import org.cbioportal.genome_nexus.service.enricher.IsoformAnnotationEnricher;
+import org.springframework.stereotype.Component;
+
+@Component
+public class IsoformAnnotationEnricherFactory {
```

Comment:
Need to add IsoformAnnotationEnricherFactory because the original IsoformAnnotationEnricher is an abstract class and not managed by Spring, so it cannot read config from application.properties, adding @Component to IsoformAnnotationEnricherFactory as this class can be managed by Spring